node/metrics.go

```diff
@@ -150,3 +169,59 @@ func (g *Metrics) AcceptBatches(status string, batchSize uint64) {
 	g.AccuBatches.WithLabelValues("number", status).Inc()
 	g.AccuBatches.WithLabelValues("size", status).Add(float64(batchSize))
 }
+
+func (g *Metrics) collectOnchainMetrics() {
+	ticker := time.NewTicker(time.Duration(uint64(g.onchainMetricsInterval)))
+	defer ticker.Stop()
+
+	// 3 chain RPC calls in each cycle.
```

Comment:
why is this 3 ? curious to know!

Reply:
It's all in the "chain RPC" log below (if that call fails), basically it needs to: 1) get current block number; 2) get the operator's quorums; 3) get the operator states.

We could by default set it to 3mins per cycle, so this metric is just adding 1 chain PRC per min.
Operators could adjust it via flag per their need.

Reply:
oh I see. makes sesne.